src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -779,10 +786,25 @@ public void getBlUsersSelf(List<String> include, boolean expandPerms,
   }
 
   @Override
-  public void postBlUsersLogin(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor,//NOSONAR
+  public void postBlUsersLoginWithExpiry(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor,
+                               LoginCredentials entity, Map<String, String> okapiHeaders,
+                               Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler,
+      Context vertxContext) {
+    doPostBlUsersLogin(expandPerms, include, userAgent, xForwardedFor, entity, okapiHeaders, asyncResultHandler, LOGIN_ENDPOINT);
+  }
+
+  @Override
+  public void postBlUsersLogin(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor,
                                LoginCredentials entity, Map<String, String> okapiHeaders,
                                Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler,
       Context vertxContext) {
+    doPostBlUsersLogin(expandPerms, include, userAgent, xForwardedFor, entity, okapiHeaders, asyncResultHandler, LOGIN_ENDPOINT_LEGACY);
+  }
+
+  private void doPostBlUsersLogin(boolean expandPerms, List<String> include, String userAgent, String xForwardedFor, //NOSONAR
+                              LoginCredentials entity, Map<String, String> okapiHeaders,
+                              Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler,
+                              String loginEndpoint) {
```

Comment:
Please format these method signatures consistently. 

Either line up each subsequent line with the start of the parameters or indent using the standard length only. My preference is for the latter because it requires less ongoing maintenance and wastes less horizontal real estate.

Reply:
Sure no problem.